### DISCUSSION TITLE
SVG

### DISCUSSION THREAD

**Discussion by lorezyra** [xushier/HD-Icons#35] [Q&A]
Do you offer any of these icons in SVG format?

**Comment by xushier** [comment]
Some of these icons have files in SVG format. I can offer it if you need.

**Reply by lorezyra** [reply]
I strongly prefer SVG over png. This way I only need to manage one image for all platforms (desktop, tablet, smartphone).

**Comment by xushier** [comment]
For some icons that do not have svg format or the icons are skeuomorphic icons with multiple colors, I can’t make svg icons and have to use photoshop.

**Reply by lorezyra** [reply]
I use Adobe Illustrator for SVG files. Photoshop is great for rasterized images, but those don't scale. With PNG, you have to use the `<picture>` element in HTML and identify all the different sizes for each major screen size. You don't want to use a 32x32 image on a 8K screen. You have to use a different, larger image. That's a major chore. Most logos are simple and easy to manage as SVG.

**Reply by xushier** [reply]
However, the logo in some projects is originally in png format. If you directly use png to svg, the svg created in this way should not have the characteristics of the normal svg format, right? Unless you draw from scratch using Adobe Illustrator.

**Reply by lorezyra** [reply]
That's true. If I really need a logo and no SVG is available, I will have someone make it for me.

**Reply by xushier** [reply]
I also want to draw the svg myself. . .So I'm getting ready to learn Adobe Illustrator
